spigot-tests/index.ts

```diff
@@ -12,7 +12,7 @@ import stupidServer from 'stupid-server';
 import { PuppeteerManager } from './puppeteer/launch';
 import { TestConfig } from './types/test-config';
 import sdkPkg from '../package.json';
-import { v4 } from 'uuid';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
I was changing some of the other imports and decided to just unify them all to this pattern (shown in the uuid docs: https://github.com/uuidjs/uuid#only-named-exports-supported-when-using-with-nodejs-esm ), even though I'm a little unclear if `{ v4 }` is problematic.